[Docs] Update docs on data privacy risks involving use of LangSmith and LangFuse (#894)
mckinsey/vizro: vizro-ai/docs/pages/explanation/safety-in-vizro-ai.md

- Always review and understand the selected model before connecting with `vizro_ai`

To learn more, refer to the section that describes how to [safeguard execution of dynamic code](safeguard.md).

## Debugging in Vizro-AI with LangSmith and LangFuse

[LangSmith](https://docs.smith.langchain.com/) and [LangFuse](https://langfuse.com/docs) are tools designed to enhance transparency and interpretability in AI workflows.

> You can use these tools with Vizro-AI to improve debugging and traceability by leveraging their advanced observability and evaluation features.

Reviewer: ```suggestion
To improve debugging and traceability, you can use the advanced observability and evaluation features of these tools with Vizro-AI.
```

> To ensure responsible use of said tools, review their data privacy and security policies. See [LangFuse](https://langfuse.com/docs/data-security-privacy) and [LangSmith](https://www.langchain.com/privacy-policy) site for more details on responsible usage.

Reviewer: ```suggestion
To ensure responsible use, review their data privacy and security policies. See [LangFuse](https://langfuse.com/docs/data-security-privacy) and [LangSmith](https://www.langchain.com/privacy-policy) site for more details.
```
